karma/module.py

```diff
@@ -223,12 +223,18 @@ def format_emojis(emojis) -> List[str]:
                 elif type(emoji) == DiscordEmoji:
                     emoji_str = f"<:pumpkin:{emoji.emoji_id}>"
 
-                idx = i // 8
-                if i % 8 == 0:
+                idx = i // 24
+
+                if i % 8 == 0 and i % 24 != 0:
+                    emoji_lists[idx].append("\n")
+
+                if i % 24 == 0:
```

Comment:
I don't really like this but I don't know if using double nested lists is alright either.

What I came up with was this:
```py
rows = [emojis[i : i + 8] for i in range(0, len(emojis), 8)]
messages = [rows[i : i + 3] for i in range(0, len(rows), 3)]
```
which creates a `List[List[List[emoji]]]` - a list of messages containing a list of rows (up to 3 rows in a message) with up to 8 items in a row. 

from this it would be easy to construct the message without relying on the number of iterations

I don't really think is it's desirable to be using this either but this feels prettier to me I guess. Leaving this for @sinus-x to decide

Reply:
Ok, this seems better. Done.